repo_health/index/static/client/src/components/repo-details/repo-details.spec.js

```diff
@@ -13,100 +13,100 @@
 const module = angular.mock.module;
 
 describe('Repo Details', () => {
-  let $rootScope;
-  let $q;
-
-  beforeEach(module(
-    'app.resources', 
-    'components.search', 
-    'components.repo-details')
-  );
-
-  describe('RepoDetailsController', () => {
-    let $componentController;
-    let controller;
+    let $httpBackend;
+    let $apiUrl, getStatsReqUrl, getDetailsUrl;
 
+    beforeEach(module(
+        'repo-health',
+        'components.repo-details',
+    ), ($provide) => {
+        $provide.provider('StatsService', () => {
```

Comment:
I was getting an unknown provider error. I think adding the `repo-health` module fixed it. Tests pass with it so I removed them.